mappings/net/minecraft/recipe/RecipeMatcher.mapping

```diff
@@ -77,6 +78,7 @@ CLASS net/minecraft/unmapped/C_nyagoaau net/minecraft/recipe/RecipeMatcher
 		FIELD f_wkisrowc ingredientItemLookup Lit/unimi/dsi/fastutil/ints/IntList;
 		METHOD <init> (Lnet/minecraft/unmapped/C_nyagoaau;Lnet/minecraft/unmapped/C_awrmdwqd;)V
 			ARG 1 recipe
+			ARG 2 recipe
```

Comment:
this constructor has one paremeter, what is going on here?

Reply:
i attempted to look at it; what in the world

Reply:
absolutely baffled as why to why enigma decided the parameter's index is 2